navigation/navigation-common-ktx/build.gradle

```diff
@@ -37,10 +37,6 @@ android {
         }
     }
 
-    packagingOptions {
```

Comment:
Why this change? was excluding files that were needed? why we needed before/was working before?

Reply:
To be honest, I don't remember why I added that on the gradle files, but with that lines the code from the ktx artifacts is not visible.